doc/man/8/rados.rst

```diff
@@ -106,8 +106,8 @@ Pool specific commands
 :command:`listwatchers` *name*
   List the watchers of object name.
 
-:command:`ls` *outfile*
-  List objects in given pool and write to outfile.
+:command:`ls` *outfile* --regex *regular expression*
```

Comment:
please add square brackets to note that `--regex` option is optional. please don't add space in the metavar, `--regex *pattern*` is good enough